### PR TITLE
docs: Add airthings wave plus docs

### PR DESCRIPTION
**Pull request by jeromelaban**
## Description:

Adds documentation the AirThings device scanner and Wave Plus sensors.

**Related issue (if applicable):** https://github.com/esphome/feature-requests/issues/147

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** https://github.com/esphome/esphome/pull/1656

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/airthings_ble.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    :width: 60.0%
+
+Configuration example:
+
+.. code-block:: yaml
+
+    sensor:
+      - platform: airthings_wave_plus
+        mac_address: '01:02:03:04:05:06'
+        update_interval: 5min # default
```
Comment (sylvinus) [inline components/sensor/airthings_ble.rst:55]: The `update_interval` option doesn't seem to be supported using develop/jela/waveplus@38dd289ea2fa5826d6c7c84ee98b1fcab5556a6e :

```
INFO Reading configuration detect.yaml...
Failed config

sensor.airthings_wave_plus: [source detect.yaml:13]
  platform: airthings_wave_plus
  mac_address: XX:XX:XX:XX:XX:XX

  [update_interval] is an invalid option for [sensor.airthings_wave_plus]. Please check the indentation.
  update_interval: 5min [source detect.yaml:15]
  temperature:
    name: WavePlus Temperature
  radon:
    name: WavePlus Radon
  radon_long_term:
    name: WavePlus Radon Long Term
...
```

Comment (jeromelaban) [reply components/sensor/airthings_ble.rst:55]: Thanks for the review! This is fixed in https://github.com/esphome/esphome/pull/1656/commits/045a72d4c5350f6aa96ae3eb870ef1a23e6a7c8f

File: index.rst [hunk 2]
```diff
@@ -250,6 +251,7 @@ Environmental
 .. imgtable::
 
     AHT10, components/sensor/aht10, aht10.jpg, Temperature & Humidity
+    AirThings BLE, components/sensor/airthings_ble, airthings_logo.png, Radon, Temperature & Humidity & Pressure
```
Comment (jesserockz) [inline index.rst]:
```suggestion
    AirThings BLE, components/sensor/airthings_ble, airthings_logo.png, Temperature & Humidity & Pressure
```

**Review by jesserockz — APPROVED** [review]

File: index.rst (hunk 2 above)
Comment (jeromelaban) [reply index.rst]: Radon reading really is supported, even if the class is not there.

Comment (jesserockz) [reply index.rst]: Ah, Airthings is in the home page in two places, `Air Quality` and `Environment`. I was not too sure which one Radon should come under, and I put it under the `Air Quality` header as it was quite long under `Environment`... 

Feel free to make another PR to update the specific classifications. =)

**Pull request merged**